UnityProject/Assets/Scripts/Chemistry/Effects/Item.cs

```diff
@@ -7,11 +7,23 @@ namespace Chemistry.Effects
 	[Serializable]
 	public class Item : Chemistry.Effect
 	{
+		public RegisterPlayer playerHolding;
 		public GameObject spawnItem;
 		public override void Apply(MonoBehaviour sender, float amount)
 		{
 			amount = (int)Math.Floor(amount);
-			Spawn.ServerPrefab(spawnItem, sender.gameObject.RegisterTile().WorldPositionServer, null, null, (int)amount);
+			if (sender.gameObject.GetComponent<MixingBowl>() != null)
```

Comment:
cache `sender.gameObject` before `if`.

Reply:
then check if the `sender.gameObject` is null.
then cache `GetComponent<MixingBowl>()` and check if it's null.

Reply:
the sender's gameobject will never be null, since the sender is always attached to a gameobject